local_dev_scripts/device-deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+source ./device-build.config
+
+set +e
+echo "uninstalling ..."
+adb uninstall ch.opengis.qfield
```

Comment:
This will clean any configuration. Maybe this should go with a `-f` (force) option and `adb install -r` should be default?